changelogs/current.yaml

```diff
@@ -8,6 +8,9 @@ minor_behavior_changes:
 - area: adaptive concurrency filter stats
   change: |
     Multiply the gradient value stat by 1000 to make it more granular (values will range between 500 and 2000).
+- area: QUIC
+  change: |
+    Port migration is default turned off.
```

Comment:
Can you add more details to this, such as that this affects QUIC clients, what the implications may be, and link to the config knob for overriding the default?

Reply:
done